tests/vmi_multus_test.go

```diff
@@ -153,68 +149,71 @@ var _ = Describe("Multus", func() {
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	AfterEach(func() {
+		// Multus tests need to ensure that old VMIs are gone
```

Comment:
There was a lot of extra handling for cleanig VMIs happening. I removed all these extra `AfterEach` and simply wipe all VMIs in both test namespaces and wait until no VMIs are left in a global AfterEach. That should give the same result.